package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "babel-upgrade",
-  "version": "0.0.15",
+  "version": "0.0.16",
```

Comment:
Typically version numbers should never be bumped in PRs (or at least, not in PRs th do anything else)

Reply:
Forgot to rebase before pushing, the master branch actually has version as `0.0.16` not sure why the diff isn't updated with `master`.

Reply:
ah i was just looking at the "view changes" diff, no worries